app/controllers/comments_controller.rb

```diff
@@ -221,7 +221,12 @@ def hide
     authorize @comment
     @comment.hidden_by_commentable_user = true
     @comment&.commentable&.update_column(:any_comments_hidden, true)
-    if @comment.save
+
+    @comment_notification = Notification.find_by(notifiable_type: "Comment",
+                                                 user_id: current_user.id,
+                                                 notifiable_id: params[:comment_id])
+
+    if @comment.save && @comment_notification.destroy
```

Comment:
I just realized that I need to handle the case where the notification does not exist anymore.